.travis.yml

```diff
@@ -11,18 +11,19 @@ notifications:
       - travis-build@platform-notifications.robomaker.aws.a2z.com
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=.rosinstall.master
+    - GITHUB_ACCOUNT=aws-robotics
+    # uncomment PACKAGE_NAME to build unit tests - REQUIRED if you'd like to run unit tests using colcon
+    - PACKAGE_NAME=cloudwatch_logger
   matrix:
-    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+   # define DOWNSTREAM_GITHUB_REPO=name-of-your-repo if you'd like to trigger downstream repos' travis builds
+    - ROS_DISTRO=kinetic ROS_VERSION=1
+    - ROS_DISTRO=lunar   ROS_VERSION=1
+    - ROS_DISTRO=melodic ROS_VERSION=1
 matrix:
   allow_failures:
-    - env: ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - env: ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - env: ROS_DISTRO=lunar   ROS_VERSION=1
+    - env: ROS_DISTRO=melodic ROS_VERSION=1
```

Comment:
master branch supports melodic right? Should we be allowing failures?

Reply:
You are right, I have removed that so only lunar failures are allowed